facter4blog.md

```diff
@@ -0,0 +1,111 @@
+# Facter 4: back to the roots
+
+Facter 4 is the default facter staring with Puppet 7. Facter 4 aims for 100% API compatible with Facter 3, but there may be some small inconsistencies. It is written in Ruby, and retains many of the same performance improvements that Facter 3 has over Facter 2. In addition to this Facter 4 also introduces granular blocking and caching for all types of facts, user defined fact groups that can be used for blocking and caching, custom and external fact hierarchies via the dot notation and profiling via `--timing` option
```

Comment:
Facter 4 aims for 100% API compatible with Facter 3, but there may be some small inconsistencies. ➡️ Facter 4 aims to be 100% API compatible with Facter 3, but there may be some small inconsistencies.

i highly recommend putting one sentence per line, otherwise reviews are painful

Reply:
Have you tried "Insert a suggestion <cmd-g>"?

```suggestion
Facter 4 is the default facter staring with Puppet 7. Facter 4 aims to be 100% API compatible with Facter 3, but there may be some small inconsistencies. It is written in Ruby, and retains many of the same performance improvements that Facter 3 has over Facter 2. In addition to this Facter 4 also introduces granular blocking and caching for all types of facts, user defined fact groups that can be used for blocking and caching, custom and external fact hierarchies via the dot notation and profiling via `--timing` option
```